### PR TITLE
rpm: added openresty-pcre and openresty-zlib packages.

### PR DESCRIPTION
**Pull request by agentzh**
Also made openresty-openssl use openresty-zlib instead of the system one.

This PR supersedes #1.

@zxcvbn4038 Please review this PR for me. I've made various changes in your #1. Thanks!

### CODE REVIEW

**Comment by zxcvbn4038** [comment]
@agentzh At first glance this looks good - tomorrow I'll build everything and check out the packages.

File: rpm/SPECS/openresty-pcre.spec [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+%files devel
+%defattr(-,root,root,-)
+%{pcre_prefix}/lib/*.a
+%{pcre_prefix}/include/*.h
+
+
+%changelog
+* Wed Aug 10 2016 makerpm
+- initial build for pcre 8.3.9
```
Comment (zxcvbn4038) [inline rpm/SPECS/openresty-pcre.spec]: Need to rewrite history here, sorry about that

**Comment by zxcvbn4038** [comment]
@agentzh Ok, I built all of the packages on a Redhat 5 system everything looks good. I made one comment about a cosmetic item in the pcre spec, but otherwise I think this is good to go.

**Comment by agentzh** [comment]
@zxcvbn4038 Updated the changelog in the spec files. Thanks for your feedback and testing!

**Comment by agentzh** [comment]
Merged. Thanks!

**Pull request merged**